packages/owl-bot/src/copy-code.ts

```diff
@@ -649,11 +646,13 @@ export async function regeneratePullRequest(
 
   // Update the PR body with the full commit history.
   const {title, body} = resplit(commitMsg, WithRegenerateCheckbox.Yes);
+
+  const apiList = apiNames.length > 3 ? 'Many APIs' : apiNames.join(',');
```

Comment:
A small nit, but is it worth having a helper function for creating apiList?

Reply:
I thought about it, because the same line of code appears twice.  If it appears a third time, I'll wrap it in a function.